capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -117,6 +124,16 @@ class ClubPage extends Component {
         .catch(e => console.log(e));
   }
 
+  deleteMeeting = (meetingID) => {
+    let meetings = this.state.meetings;
+    let meetingIDsArray = this.state.meetings.map(m => m.id);
```

Comment:
nit: `const` for these since they aren't reassigned

Reply:
Fixed!